### ISSUE TITLE
As an end-user, TOTP reset process is too onerous (and for support too...)

### ISSUE THREAD

**Issue by pburkholder** [cloud-gov/product#1332]
In order to support users who have a new device, or have lost their TOTP device, we need a process to get them working again with a "sufficient" amount of verification, and less burden on support.

[need more description of current problem]

Team consensus is that stripping all roles may be overkill, and might be better to have better identity validation.

## Security considerations

There are _significant_ considerations here, as we should be resistant to social engineering this process.

## Implementation sketch/ideas:

Validation for TOTP reset could be:

* phone calls to the phone on file for an org manager (not currently gathering phone number as a recovery factor)
* email from another team member in the same org
* demonstrate control of the old device by logging in to cloud.gov and taking an auditable action

**Comment by karareinsel** [comment]
@bengerman13 and @spgreenberg is this card sufficient to get started on the self-service TOTP reset? If not, feel free to edit or create a new card.

**Comment by hillaryj** [comment]
Related issue: #1317

**Comment by spgreenberg** [comment]
Status: shibboleth id deployed in dev, but there is still a configuration issue making the cg IDP not functional. I am tracking this down.

**Comment by bengerman13** [comment]
I *think* this is just waiting on cloud-gov/cg-uaa-extras#152
